img_checker.gemspec

```diff
@@ -9,4 +9,6 @@ Gem::Specification.new do |s|
   s.files       = ["lib/img_checker.rb"]
   s.homepage    = 'http://rubygems.org/gems/ImageSizeLinter'
   s.license     = 'GPL-3.0'
+  s.add_runtime_dependency "fastimage", [">= 2.0.0"]
+  s.add_development_dependency "fastimage", [">= 2.0.0"]
```

Comment:
@Abhi2424shek You don't need to add both the development as well as runtime dependency. Just including this as a runtime dependency would do. Also add the line `gemspec` at the end of your Gemfile. May read this [blog post](http://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/) to  understand the the significance of `gemspec` method in a Gemfile.